package.json

```diff
@@ -2,22 +2,32 @@
   "name": "notification",
   "version": "6.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "webpack.mix.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "development": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+    "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --no-progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+    "dev": "npm run development",
+    "watch": "npm run development -- --watch",
+    "prod": "npm run production"
   },
   "author": "BracketSpace",
   "license": "ISC",
+  "browserslist": [
+    "chrome 60",
+    "firefox 55",
+    "safari 10",
+    "edge 15",
+    "ie 11"
+  ],
   "devDependencies": {
-    "browser-sync": "^2.26.3",
-    "gulp": "^3.9.0",
```

Comment:
`gulpfile.js` needs to be removed

Reply:
And it's removed :)

Reply:
Nope, still within the files. Check the changelog